automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -346,4 +352,29 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     val preemptibleNodePrefix = clusterName.string + "-sw"
     notebookPage.executeCell(s"! grep Finished ~/hail.log | grep $preemptibleNodePrefix").get should include(preemptibleNodePrefix)
   }
+
+  def uploadDownloadTest(cluster: Cluster, uploadFile: File, timeout: FiniteDuration)(assertion: (File, File) => Any)(implicit webDriver: WebDriver, token: AuthToken): Any = {
+    uploadFile.exists() shouldBe true
```

Comment:
I like this quick check.  Might as well add a `cluster.status() shouldBe ClusterStatus.Running` too.